keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/SecurityGroupHandler.kt

```diff
@@ -53,160 +57,190 @@ class SecurityGroupHandler(
   private val orcaService: OrcaService,
   override val objectMapper: ObjectMapper,
   override val normalizers: List<ResourceNormalizer<*>>
-) : ResourceHandler<SecurityGroup> {
+) : ResolvableResourceHandler<SecurityGroupSpec, Map<String, SecurityGroup>> {
```

Comment:
👍 I'm gonna change these interface names to encourage us to use this one more often.

Reply:
See #491 (I won't merge until after this goes in)